addons/event/models/__init__.py

```diff
@@ -9,3 +9,4 @@
 from . import event_ticket
 from . import res_config_settings
 from . import res_partner
+from . import mail_template
```

Comment:
You know what "they" say about ordering :D ?

Reply:
oops